### DISCUSSION TITLE
vsh: Print output of execute() directly to stdout

### DISCUSSION THREAD

**Discussion by fossdd** [vlang/v#9220] [Ideas]
One difference from bash, sh,.. and vsh is that Commands executed, will not live printed to the terminal.

For example:
```vlang
println(execute('echo first; sleep 10; echo after').output)
```
does print the output after the command was run. But isn't there a function that prints it directly?

I don't develope programming languages or so and I don't know how that can be implemented, but I guess the idea is not bad.

Also don't replace it with the `execute()`. Adding something like `execute_and_print`.

The return value can be the same as in `execute()`.

**Comment by spytheman** [comment]
system() does what you need (unlike execute(), system() will not wait for the command to finish before showing its output/stderr)

**Reply by spytheman** [reply]
it does not capture the output though, it returns just the exit code of the passed command to the caller
